suites/reef/cephadm/tier2-cluster-elasticity.yaml

```diff
@@ -724,50 +688,27 @@ tests:
             limit: 2            # no of daemons
             sep: " "            # separator to be used for placements
       destroy-cluster: false
-      abort-on-fail: true
-
-  - test:
-      name: Scale down NFS Service
-      desc: Scale down NFS-Ganesha service to 1 node
-      module: test_nfs.py
-      polarion-id: CEPH-83573586
-      config:
-        command: apply
-        service: nfs
-        base_cmd_args:
-          verbose: true
-        pos_args:
-          - mynfs                         # nfs service ID
-          - nfs-ganesha-pool              # name of the pool
-        args:
-          placement:
-            nodes:
-              - node10
-            limit: 2
-            sep: ";"
-      destroy-cluster: false
-      abort-on-fail: true
 
-  - test:
```

Comment:
why this test case is skipped ?

Reply:
rbd-io was throwing socket timeout at times. so commented it